app/flatpak-builtins-info.c

```diff
@@ -153,9 +153,7 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
 
   if (opt_show_commit)
     {
-      if (first)
-        first = FALSE;
-      else
+      if (!first)
```

Comment:
We don't want to remove this because if we later add a new option when first would not be unset if we make this change.